test/gpu/octree/perfomance.cpp

```diff
@@ -98,7 +98,7 @@ TEST(PCL_OctreeGPU, performance)
 
     //prepare host cloud
     pcl::PointCloud<pcl::PointXYZ>::Ptr cloud_host(new pcl::PointCloud<pcl::PointXYZ>);	
-    cloud_host->width = data.size();
+    cloud_host->width = data.points.size();
     cloud_host->height = 1;
     cloud_host->points.resize (cloud_host->width * cloud_host->height);    
```

Comment:
Should we adress this as well before merge? Removal of ->points here etc.

Reply:
I thought `points` was added because `data` doesn't have `size` function

Reply:
other PR. this is blocking haritha's work and is a minor thing.

Reply:
`data.point`s has to be there, but `cloud_host->points.resize() `is ordinary cloud object.

I'll make a followup :+1: